.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+npm-debug.log
+.DS_Store
+dist
```

Comment:
can we add ide setup files to this list too, I have a nice gitignore file if you like :)